mantidproject/mantid
# Show fitted parameter plot

Fixes #12922.

**To test**
1. Add two or more spectra to the Multi-dataset interface. Ideally those should be form different runs of the same instrument and have some logs with different values. 
2. Define a function but don't set any global parameters.
3. In the property browser below the function set "Fitting" type to "Sequential".
4. Set some LogValue and Plot parameter and bottom of the browser.
5. Fit
6. A plot of "Plot parameter" against "LogValue" should show. It must have error bars.
